Update user-research.md (#365)
BitcoinDesign/Guide: guide/designing-products/user-research.md

# User research

> We design products to benefit users. That’s why it is necessary to understand their needs, desires, and behaviors.

Reviewer: I think this removes the important point that design is not solely about users. It is often about balancing user needs with those other aspects (tech, business model, etc). Bitcoin is a prime example, as the tech is highly opinionated and forces users to change certain behaviors.

Author: Fixed in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

   layout = "float-left"
%}

**User research helps teams efficiently create better and more focused products**

Prioritizing features that make the most significant difference is crucial. Those features are seldom apparent, and their discovery is often the result of debate and critical thinking. If you find yourself stuck, try making data-informed decisions.

> The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, in which many open-source efforts are.

Reviewer: Removes a point I was trying to explicitly make about being too focused on the tech while not really addressing user needs. It was meant to make the reader question whether this is something they are "guilty of".

Author: Reverted in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

> The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, in which many open-source efforts are.

Reviewer: ```suggestion
The results are appropriate product roadmaps, better-focused team efforts, and happier users. This is especially important for projects with limited resources, which many open-source efforts are.
```

Author: I don't think this sounds grammatically correct.

   layout = "float-right-desktop"
%}

When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate it.

> Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible with Bitcoin, and what was needed from Bitcoin.

Reviewer 2: ```suggestion
Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible, and what was needed from Bitcoin.
```

Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.

- [Bitcoin on Github](https://github.com/bitcoin)
- [Bitcoin mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)

> - [Bitcointalk](https://bitcointalk.org/)

Reviewer 2: ```suggestion
- [Bitcointalk](https://bitcointalk.org/)
- [Bitcoin subreddit](https://www.reddit.com/r/Bitcoin/)
- [Bitcoin Core dev meeting logs](http://www.erisian.com.au/meetbot/bitcoin-core-dev/)
```

Two suggestions for some archives which have a great deal of resources hidden within.

   layout = "float-right-desktop"
%}

> When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate it.

Reviewer: Are you sure you meant to use `proliferate` here?

Author: Reproduce rapidly, spread, so yeah it makes sense to me, or do you have another better-suited word in mind?

Reviewer: It just changes the original point. This new version states that Bitcoin got quickly adopted by english-speaking audiences because it was first released in English. The original point was that Bitcoin is not limited by context (local laws, culture, borders...), which allowed it to have a diverse user base right away (meaning people all over the world could use it) - despite the original limitation of being English-only. That builds on the argument of this paragraph that researchers need to think differently about their users - global and diverse from the start vs start local and become global. Does that make sense?

Reviewer 2: Agree with @GBKS, doesn't read as clearly or tie in as nicely with the rest of the paragraph.

Author: @GBKS @Bosch-0 Reverted in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a

> Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible, and what was needed from Bitcoin.

Reviewer: Not sure why this is written in the past now. These communities are still active and super helpful for user research today.

Author: Good point.

Author: @GBKS 

> Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.

The later paragraph though explains that these are still available, what do you suggest we do here?

Reviewer: > These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.

I think we should keep the point being made here. Lots of designers don't realize that a great source of user feedback is so easily available to them. They look for formal research documents, when all they need to do is to tap into the online communities that are just a click away. It's something that I'd like to stress further in future discussions of user research.

Let me try to combine the brevity of the rewrite with the points I think we should make.

> Diverse online communities rapidly formed around Bitcoin. Vast amounts of public conversation can help designers and developers comprehend what is possible, and what is needed from Bitcoin. These communities are great for brainstorming ideas, getting feedback, and involving users actively in the design process.

And then I'd remove the next sentence, which is this one:

> Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.

Author: @GBKS  Addressed in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a

Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.

- [Bitcoin on Github](https://github.com/bitcoin)
- [Bitcoin mailing list](https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-dev)
- [Bitcointalk](https://bitcointalk.org/)

> - [Bitcoin subreddit](https://www.reddit.com/r/Bitcoin/)

Reviewer: Subreddit is listed twice.

Author: ```suggestion
```

- [Bitcoin Core dev meeting logs](http://www.erisian.com.au/meetbot/bitcoin-core-dev/)
- [Bitcoin subreddit](https://www.reddit.com/r/bitcoin)

</div>

### An unknowable userbase

> Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While Bitcoin's privacy provides is a fundamental aspect, it can make things challenging for user researchers.

Reviewer: ```suggestion
Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While the privacy Bitcoin provides is a fundamental aspect, it can make things challenging for user researchers.
```
Fixed the sentence. Still reads a bit funny though.

### An unknowable userbase

> The Bitcoin protocol only knows its users through pseudonymous addresses. We can analyze the Bitcoin blockchain for fundamental transaction behavior analysis, but further analysis of addresses and transactions can be complicated. While this provides incredible privacy and security for users, it can make things challenging for user researchers.

Reviewer 2: ```suggestion
Bitcoin only knows its users through pseudonymous addresses. We can analyze on-chain metrics for things like fundamental transaction behaviors, but analysis of addresses and transactions beyond this can be complicated. While Bitcoin's privacy provides is a fundamental aspect, it can make things challenging for user researchers.
```

   layout = "float-left"
%}

> While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate database of transactions. Here’s how we can use this data to improve bitcoin for users:

Reviewer 2: ```suggestion
While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. Here’s how we can use this data to improve Bitcoin for users:
```

Reviewer 2: There isn't examples listed after the last statement which implies there is some.

   layout = "float-left"
%}

> While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. Here’s how we can use this data to improve Bitcoin for users:

Reviewer: > Here’s how we can use this data to improve Bitcoin for users:

This does not make sense as the following text is a new topic and not related. The original sentence was a question meant to inspire the reader to think about new ways to use the public ledger data to create better experiences.

Author: What do you suggest we do with it, remove it or?

Author: Found a good middle ground in https://github.com/BitcoinDesign/Guide/pull/365/commits/7a005dfb2b53e00b399f33ff33963a3f2a505a3d

Reviewer: How about this? Makes it a little more clear that it's an appeal to the reader.

> As designers, what can we learn from this data that allows us to improve the experience of our users?

Author: Added in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a not sure if that's what you meant.

   layout = "float-right-desktop"
%}

New technologies can take decades to reach mainstream adoption. Bitcoin is, in our opinion, still on its way up the adoption curve, which means it and we are early. This affects research in multiple ways. Early adopters have different use cases and needs than users who adopt Bitcoin ten years from now. Products designed today could be outdated by the time that the early majority embraces Bitcoin.

> Secondly, as Bitcoin matures, its capabilities will evolve. Scaling and performance improvements will make complex transactions more affordable, private, and secure. These properties will make early and later adopters more comfortable embracing bitcoin both as a store of value and as a medium of exchanges.

Reviewer 2: ```suggestion
Secondly, as Bitcoin matures, its capabilities will evolve. Scaling and performance improvements will make transactions more affordable, private, and secure. These improvements will make people more comfortable embracing bitcoin as both a store of value and medium of exchange.
```

   layout = "float-left"
%}

> User research should not be a one-sided extraction of collaboration between researchers and users. Since researchers initiate research efforts, it is up to them to create frameworks that respect users’ privacy. That means upfront disclosures, open collaboration, and transparency about how research results are used.

Reviewer: ```suggestion
User research should not be a one-sided extraction of information, but a collaboration between researchers and users. Since researchers initiate research efforts, it is up to them to create frameworks that respect users’ privacy. That means upfront disclosures, open collaboration, and transparency about how research results are used.
```
Fixed broken sentence.
